forge/src/main/java/org/openjdk/skara/forge/PullRequest.java

```diff
@@ -181,7 +181,8 @@ default boolean isSame(PullRequest other) {
     }
 
     /**
-     * Get the last force-push time. If there is no force-push in pull request
+     * Return the last time something was force pushed while not in draft state.
+     * If there is no force-push in pull request
      * or the restful api doesn't support force-push, return empty.
```

Comment:
While this is strictly less than 80, we also like to keep the lines even. I'm not sure if Intellij has any automation for this. I've just capitulated to doing it manually.

Reply:
I also don't know whether IntelliJ can do this. I will do it manually.